lib/ohai/system.rb

```diff
@@ -69,6 +68,7 @@ def reset_system
 
       @loader = Ohai::Loader.new(self)
       @runner = Ohai::Runner.new(self, true)
+      @runner.backend = backend unless @backend
```

Comment:
Why not just `@runner.backend = backend`?

Reply:
I think this was related to Chef Target mode. When you start Ohai standalone, it's ok because it creates a remote backend in that case. If you use Chef TM, there was some Ohai reset call which would overwrite the pre-initialized backend passed by Chef with a new, local one. I'd have to test if this holds true or if my memory is off.